src/v/cluster/archival/ntp_archiver_service.cc

```diff
@@ -1796,12 +1796,7 @@ ss::future<ntp_archiver_upload_result> ntp_archiver::upload_segment(
     // the read path will create the index on the fly while downloading the
     // segment, so it is okay to ignore the index upload failure, we still
     // want to advance the offsets because the segment did get uploaded.
-
-    ssx::spawn_with_gate(
-      _gate,
```

Comment:
>It's unclear what affect this has and needs more analysis.
>We'll put it back in the foreground for now and assess viability of
>backgrounding at a later time.

We have some suspicion that there is a negative impact associated with backgrounding in some edge cases. Mind sharing that in the commit message?

Reply:
done